### PR TITLE
Refs task 6341: Init project

### PR DESCRIPTION
**Pull request by trieuthan92pro**
Feature : init project
Link: https://edu-redmine.framgia.vn/issues/6341

### CODE REVIEW

**Comment by thantrieu** [comment]
> https://edu-redmine.framgia.vn/issues/6341

**Comment by trieuthan92pro** [comment]
Ready

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Hello World!"
```
Comment (DoanVanToan) [inline app/src/main/res/layout/activity_main.xml]: remove hardcode
all string have to defined in string resource

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+</android.support.constraint.ConstraintLayout>
```
Comment (DoanVanToan) [inline app/src/main/res/layout/activity_main.xml]: add end of file

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+    <background android:drawable="@drawable/ic_launcher_background" />
+    <foreground android:drawable="@drawable/ic_launcher_foreground" />
+</adaptive-icon>
```
Comment (DoanVanToan) [inline app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml]: add end of file

**Comment by trieuthan92pro** [comment]
Ready

**Review by DoanVanToan — APPROVED** [review]

**Pull request merged**